lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -916,6 +937,10 @@ def _create_deployment_parameters(
 
             # Set disk type
             azure_node_runbook.disk_id = AzurePlatform._get_disk_id(node_space.features)
+            assert isinstance(node_space.data_disk_iops, int)
+            azure_node_runbook.data_disk_size = get_data_disk_size(
```

Comment:
It should be resolved in `_node_generate_min_capability`. What's the reason to move it to here?

Reply:
Current code logs meet the requirement, I tested with different IOPS.

Reply:
Anyway, I will handle it in feature's refactoring

Reply:
Let's sync up again tomorrow, data_disk_iops is always IntRange here, then get the min value of IOPS combined with disk type when data disk count more than 0 is current approach.